x/src/lint/engine/runner.rs

```diff
@@ -0,0 +1,194 @@
+// Copyright (c) The Libra Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::lint::engine::{prelude::*, LintContext};
+use once_cell::sync::OnceCell;
+use std::{
+    ffi::OsStr,
+    path::Path,
+    process::{Command, Stdio},
+};
+
+/// Configuration for the lint engine.
+#[derive(Clone, Debug)]
+pub struct LintEngineConfig<'cfg> {
+    project_root: &'cfg Path,
+    content_linters: Option<&'cfg [&'cfg (dyn ContentLinter + 'cfg)]>,
```

Comment:
How many times can you put `'cfg` on the same line 😉

Reply:
it's all covariant so it'll work out ;)